extension/src/popup/components/manageNetwork/NetworkForm/index.tsx

```diff
@@ -108,7 +265,22 @@ export const NetworkForm = ({ isEditing }: NetworkFormProps) => {
               placeholder={t("Enter passphrase")}
             />
             {isEditing ? (
-              "Remove"
+              <div className="NetworkForm__remove-wrapper">
+                {!isMainnetOrTestnet && (
```

Comment:
Disabling the `remove` button if someone tries to edit the Mainnet or Testnet config